CorsixTH/Lua/ui.lua

```diff
@@ -720,6 +723,11 @@ function UI:toggleFullscreen()
     -- Save new setting in config
     self.app.config.fullscreen = self.app.fullscreen
     self.app:saveConfig()
+    -- If options window is open, make sure the button updates on hotkey
+    if not from_uioptions and self:getWindow(UIOptions) then
```

Comment:
The second param of the if is more a safeguard, but happy to remove it if not needed.

Reply:
This is a bit of a violation again, you shouldn't care where a function was called from generally.

window:updateFSButtonOnHotkey() should be in the appropriate caller instead.